tests/functional/adapter/materialized_view_tests/utils.py

```diff
@@ -67,3 +68,20 @@ def query_autorefresh(project, relation: RedshiftRelation) -> bool:
         and trim(mv.db_name) ilike '{ relation.database }'
     """
     return project.run_sql(sql, fetch="one")[0]
+
+
+def run_dbt_and_capture_with_retries(args: List[str], max_retries: int = 10):
```

Comment:
since we're adding this in the mv scope it'll probably help folks coming along later if the function name indicates it's defined here and not in core

Reply:
When you say "here", are you concerned more about Redshift, MVs, or both?

Reply:
hmm I guess technically it _should_ be both but that seems kinda unwieldy, how about:
```suggestion
def mv_run_dbt_and_capture_with_retries(args: List[str], max_retries: int = 10):
```